src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.Windows.cs

```diff
@@ -1219,7 +1223,11 @@ private void FinishOperationSendPackets()
 
         private static readonly unsafe IOCompletionCallback s_completionPortCallback = delegate (uint errorCode, uint numBytes, NativeOverlapped* nativeOverlapped)
         {
-            var saeaBox = (StrongBox<SocketAsyncEventArgs>)ThreadPoolBoundHandle.GetNativeOverlappedState(nativeOverlapped)!;
+#pragma warning disable CA1416 // https://github.com/dotnet/roslyn-analyzers/issues/4090
```

Comment:
https://github.com/dotnet/roslyn-analyzers/issues/4090 was resolved.  Is the `#pragma warning disable` still required? If we don't yet have a build of the analyzer with this, we might want to just wait on this PR until we do.

Reply:
I believe it is built to https://dev.azure.com/dnceng/public/_packaging?_a=package&feed=dotnet5&view=versions&package=Microsoft.CodeAnalysis.NetAnalyzers&version=5.0.0-rc2.20453.2&protocolType=NuGet, we need to update the dogfood version https://github.com/dotnet/runtime/blob/master/eng/Analyzers.props#L9